File: app/src/main/java/com/fjun/hassalarm/AboutActivity.java
package com.fjun.hassalarm;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.fjun.hassalarm.databinding.ActivityAboutBinding;

public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        final ActivityAboutBinding binding = ActivityAboutBinding.inflate(getLayoutInflater());
        setContentView(binding.root);
    }

    public static Intent createIntent(Context context) {
        return new Intent(context, AboutActivity.class);
    }
}

File: app/src/main/java/com/fjun/hassalarm/AboutActivity.kt
package com.fjun.hassalarm

import android.content.Context
import android.content.Intent
import android.os.Bundle
import androidx.activity.compose.setContent
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import com.fjun.hassalarm.ui.theme.HassAlarmTheme

class AboutActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContent {
            HassAlarmTheme {
                AboutScreen()
            }
        }
    }

    companion object {
        fun createIntent(context: Context?): Intent = Intent(context, AboutActivity::class.java)
    }
}